types/mdast/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-redundant-undefined": false
```

Comment:
Is this needed?

Reply:
@elibarzilay see https://github.com/DefinitelyTyped/DefinitelyTyped/pull/54421#discussion_r667340048.
I believed TS folks were adding explicit undefineds?